stable/shared/file-assertions.yaml

```diff
@@ -0,0 +1,105 @@
+ceph-mon:
+  files:
+    - path: /etc/ceph/ceph.conf
+      owner: root
+      group: root
+      mode: '777'
+    - path: /var/lib/charm/ceph-mon/ceph.conf # /etc/alternatives/ceph.conf -> /etc/ceph/ceph.conf destination
+      owner: root
+      group: root
+      mode: '644'
+    - path: /etc/ceph/ceph.client.admin.keyring
```

Comment:
Shouldn't we be looking for a mode of 600?

Reply:
the default parameters in the checks are root:root, mode 600, so an undefined one will get 600 :)